docker/config/my.env.dist

```diff
@@ -0,0 +1,19 @@
+# This file is for settings that are specific to your docker local development
+# environment. Since these are specific to your local development environment,
+# don't check them in.
+
+# ---------------------------------------------
+# Webapp Google auth settings
+# ---------------------------------------------
+
+# Google auth credentials from setting up a project
+OAUTH2_CLIENT_ID=
+OAUTH2_CLIENT_SECRET=
+
```

Comment:
I want one file that covers all the components. I talk about why I did it this way in the other PR comments already. After reading those, I'm game for discussing it further.